lib/validation.js

```diff
@@ -277,6 +277,28 @@ module.exports = function (yargs, usage, y18n) {
     }
   }
 
+  var conflicting = {}
+  self.conflicts = function (key1, key2) {
+    if (typeof key1 === 'string' && typeof key2 === 'string' && key1 !== '_' && key2 !== '_') {
+      conflicting[key1] = key2
```

Comment:
`implies` can accept an object rather than two keys, and will iterate over the object setting multiple implications at the same time; I think `conflicts` should have the same API 👍